Automattic/wp-calypso
# Checkout: Show full "renewal" TOS for domain transfers

## Proposed Changes

Checkout displays a Terms of Service at the bottom of the page that includes the wording `You agree to our Terms of Service and authorize your payment method to be charged on a recurring basis until you cancel`. This TOS is only displayed for recurring purchases and not for one-time purchases. For one-time purchases there's a different line item that just reads `You agree to our Terms of Service`.

However, Domain Transfer products are special. Even though they are one-time purchases, they result in a Domain Registration product being create (with the same price as the Transfer costs at checkout). So in a way they are actually a Domain Registration and should have the full TOS for renewable products.

This PR updates the TOS to show the "recurring" text for domain transfers.

Inspired by this Slack thread: p1687542902188289-slack-C05CT832K2T

## Testing Instructions

- Add a domain transfer to your cart. It's not clear to me how to do this. 🤔 
- Complete each step to reach the bottom of checkout. Verify that you see the TOS with the `recurring basis` text.
